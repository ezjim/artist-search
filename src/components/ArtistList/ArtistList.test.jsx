import React from 'react';
import { shallow } from 'enzyme';
import ArtistsList from './ArtistList';

describe('ArtistsList', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<ArtistsList artists={[{ name: 'Joyner Lucas' }, { name: 'Revenge' }]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
