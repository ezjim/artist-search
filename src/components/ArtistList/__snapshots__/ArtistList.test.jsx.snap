// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistsList matches a snapshot 1`] = `
<section>
  <ul>
    <li>
      <h3>
        Electric Light Orchestra
      </h3>
      <h3 />
    </li>
    <li>
      <h3>
        Pretty Lights
      </h3>
      <h3 />
    </li>
  </ul>
</section>
`;
